testsuites/syncgateway/functional/tests/test_rollback.py

```diff
@@ -117,30 +113,20 @@ def test_rollback_server_reset(params_from_base_test_setup, sg_conf_name):
     # Delete vbucket and restart server
     cb_server.delete_vbucket(66, "data-bucket")
     cb_server.restart()
-
-    max_retries = 20
+    max_retries = 50
     count = 0
     while count != max_retries:
         # Try to get changes, sync gateway should be able to recover and return changes
-        try:
-            # A changes since=0 should now be in a rolled back state due to the data loss from the removed vbucket
-            # Seth should only see the docs not present in vbucket 66, unlike all the docs as above.
-            changes = client.get_changes(url=sg_url, db=sg_db, since=0, auth=seth_session)
+        # A changes since=0 should now be in a rolled back state due to the data loss from the removed vbucket
+        # Seth should only see the docs not present in vbucket 66, unlike all the docs as above.
+        changes = client.get_changes(url=sg_url, db=sg_db, since=0, auth=seth_session)
+        changes_ids = [change["id"] for change in changes["results"] if not change["id"].startswith("_user")]
+        log_info("length of Changes ids are {} and vbuckets {}".format(len(changes_ids), num_vbuckets))
+        if len(changes_ids) == (num_vbuckets - 1):
             break
-        except requests.exceptions.HTTPError as he:
-            log_info("{}".format(he.response.status_code))
-            log_info("Retrying in 1 sec ...")
-            time.sleep(1)
-            count += 1
+        count += 1
 
     # Make sure there are a few retries to ensure that server did go down and up
-    assert count > 1
-
+    assert count > 0
```

Comment:
Do we require at least one retry?  This seems like an unreliable assertion.

Reply:
count > 0 make sure  atleast 1 retry is done. I see that log message in the retry shows 1 retry is done and  incrementing the count to 1 and retrying.

 Test is sometimes failing with 3 nodes in cluster as it get expected changes with 1 retry , so count more than 1 does not work. Changes are getting too fast. 
Count > 1 pass with 1 node in the cluster, that is the reason I need to change it to count > 0